docs/eks-installation.md

```diff
@@ -0,0 +1,30 @@
+# Deploying Antrea in AWS EKS
+
+This document describes steps to deploy Antrea in NetworkPolicy only mode to an AWS EKS cluster.
+
+Assuming you already have an EKS cluster, and have KUBECONFIG environment variable pointing to
```

Comment:
have the `KUBECONFIG` environment variable point to?

Reply:
done